test/test_testing.py

```diff
@@ -1402,6 +1402,26 @@ def test_config_from_directory_charmcraft_yaml(self):
         self.assertEqual(harness.model.config['opt_int'], 1)
         self.assertIsInstance(harness.model.config['opt_int'], int)
 
+    def test_config_in_repl(self):
```

Comment:
@benhoyt in the past you've indicated a preference for one Harness per test. I can adjust this to do that if you'd prefer - this seems fairly contained as a logical unit to me, but I don't have strong feelings about it.

Reply:
No, this seems okay and fine to reuse the patching/etc. Nit, if you want: add a blank line before the second `harness = ...`